background/message-cache.js

```diff
@@ -57,6 +60,7 @@ export async function updateBadge(defaultStoreId) {
   // Hide badge when logged out and showOffline is false,
   // or when the logged-in user has no unread messages,
   // or when the addon is disabled
+  chrome.action.setTitle({ title: "" });
```

Comment:
Why is this new line required?

Reply:
@DNin01 the whole purpose of the tooltip was to show the message count when the badge is hidden...

Reply:
We don't want to send requests to Scratch to get the message count if the badge is hidden.

Reply:
> @DNin01 the whole purpose of the tooltip was to show the message count when the badge is hidden...

Oh.

You can revert my change if you want, or (preferably) you could change the code so that the count is only shown on the tooltip if `msg-count-badge` is enabled and Scratch Addons isn't muted (in Do Not Disturb mode).

Reply:
@iqnite Did you see this comment?

Reply:
I thought this PR was only about truncating the message count if it's a very big number.

Reply:
> I thought this PR was only about truncating the message count if it's a very big number.

It was, but apparently some people think that a tooltip that is updated together with a badge is spamming Scratch's servers and needs to be handled differently than the badge. 😕

Reply:
I'm very confused about @DNin01 's changes, I'll check back when I'm on PC (on mobile right now)